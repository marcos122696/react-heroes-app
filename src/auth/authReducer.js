import { types } from "../types/types";

// const state = {
//     name: 'Marcos',
//     logged: true
// };

// const action = {
//     type: type.login,
//     payload: {
//         name: 'Marcos',
//         email: 'marcos@gmail.com',
//     },
// };

export const authReducer = ( state = {}, action ) => {

    switch ( action.type ) {
        case types.login:
            return {
                ...action.payload, // = name: action.payload.name
                logged: true
            }
        
        case types.logout:
            return {
                logged: false
            };
    
        default:
            return state;
    }
};