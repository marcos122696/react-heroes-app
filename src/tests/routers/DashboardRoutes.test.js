import { mount } from "enzyme";
import { AuthContext } from "../../auth/authContext";
import { DashboardRoutes } from "../../routers/DashboardRoutes";
import { MemoryRouter } from "react-router-dom";


describe('Pruebas en el <DashboardRoutes />', () => {

    const contextValue = {
        user: {
            logged: true,
            name: 'Pepe'
        }
    }

    test('Debe de mosrarse correctamente - /marvel', () => {

        const wrapper = mount(
            <AuthContext.Provider value={ contextValue } >
                <MemoryRouter initialEntries={['/marvel']}>
                    <DashboardRoutes />
                </MemoryRouter>
            </AuthContext.Provider>
        );
        // console.log(wrapper.html());
        expect( wrapper ).toMatchSnapshot();
        expect( wrapper.find('.text-info').text().trim()).toBe('Pepe');
        expect( wrapper.find('h1').text().trim()).toBe('MarvelScreen');
    });

    test('Debe de mosrarse correctamente - DC', () => {

        const wrapper = mount(
            <AuthContext.Provider value={ contextValue } >
                <MemoryRouter initialEntries={['/dc']}>
                    <DashboardRoutes />
                </MemoryRouter>
            </AuthContext.Provider>
        );
        // console.log(wrapper.html());
        expect( wrapper ).toMatchSnapshot();
        expect( wrapper.find('h1').text().trim()).toBe('DcScreen');
    });
});