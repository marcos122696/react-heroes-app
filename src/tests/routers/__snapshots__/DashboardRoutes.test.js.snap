// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <DashboardRoutes /> Debe de mosrarse correctamente - /marvel 1`] = `
Array [
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <a
      className="navbar-brand ms-3"
      href="/"
      onClick={[Function]}
    >
      HeroesApp
    </a>
    <div
      className="navbar-collapse"
    >
      <div
        className="navbar-nav"
      >
        <a
          aria-current="page"
          className="nav-item nav-link active text-primary"
          href="/marvel"
          onClick={[Function]}
        >
          Marvel
        </a>
        <a
          className="nav-item nav-link "
          href="/dc"
          onClick={[Function]}
        >
          DC
        </a>
        <a
          className="nav-item nav-link "
          href="/search"
          onClick={[Function]}
        >
          Serch
        </a>
      </div>
    </div>
    <div
      className="navbar-collapse collapse w-100 order-3 dual-collapse2 d-flex justify-content-end"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <span
          className="nav-item nav-link text-info"
        >
          Pepe
        </span>
        <button
          className="nav-item nav-link btn btn-outline-secondary me-3"
          onClick={[Function]}
        >
          Logout
        </button>
      </ul>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div>
      <h1
        className="animate__animated animate__hinge"
      >
        MarvelScreen
      </h1>
      <hr />
      <div
        className="row rows-cols-1 row-cols-md-3 g-3 text-dark"
      >
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-spider"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Spider Man"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-spider.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Spider Man
                    </h5>
                    <p
                      className="card-text"
                    >
                      Peter Parker
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Amazing Fantasy #15
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-captain"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Captain America"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-captain.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Captain America
                    </h5>
                    <p
                      className="card-text"
                    >
                      Steve Rogers
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Captain America Comics #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-iron"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Iron Man"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-iron.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Iron Man
                    </h5>
                    <p
                      className="card-text"
                    >
                      Tony Stark
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Tales of Suspense #39
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-thor"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Thor"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-thor.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Thor
                    </h5>
                    <p
                      className="card-text"
                    >
                      Thor Odinson
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Journey into Myster #83
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-hulk"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Hulk"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-hulk.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Hulk
                    </h5>
                    <p
                      className="card-text"
                    >
                      Bruce Banner
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        The Incredible Hulk #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-wolverine"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Wolverine"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-wolverine.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Wolverine
                    </h5>
                    <p
                      className="card-text"
                    >
                      James Howlett
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        The Incredible Hulk #180
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-daredevil"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Daredevil"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-daredevil.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Daredevil
                    </h5>
                    <p
                      className="card-text"
                    >
                      Matthew Michael Murdock
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Daredevil #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-hawkeye"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Hawkeye"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-hawkeye.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Hawkeye
                    </h5>
                    <p
                      className="card-text"
                    >
                      Clinton Francis Barton
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Tales of Suspense #57
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-cyclops"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Cyclops"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-cyclops.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Cyclops
                    </h5>
                    <p
                      className="card-text"
                    >
                      Scott Summers
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        X-Men #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/marvel-silver"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Silver Surfer"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/marvel-silver.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Silver Surfer
                    </h5>
                    <p
                      className="card-text"
                    >
                      Norrin Radd
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        The Fantastic Four #48
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`Pruebas en el <DashboardRoutes /> Debe de mosrarse correctamente - DC 1`] = `
Array [
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <a
      className="navbar-brand ms-3"
      href="/"
      onClick={[Function]}
    >
      HeroesApp
    </a>
    <div
      className="navbar-collapse"
    >
      <div
        className="navbar-nav"
      >
        <a
          className="nav-item nav-link "
          href="/marvel"
          onClick={[Function]}
        >
          Marvel
        </a>
        <a
          aria-current="page"
          className="nav-item nav-link active text-primary"
          href="/dc"
          onClick={[Function]}
        >
          DC
        </a>
        <a
          className="nav-item nav-link "
          href="/search"
          onClick={[Function]}
        >
          Serch
        </a>
      </div>
    </div>
    <div
      className="navbar-collapse collapse w-100 order-3 dual-collapse2 d-flex justify-content-end"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <span
          className="nav-item nav-link text-info"
        >
          Pepe
        </span>
        <button
          className="nav-item nav-link btn btn-outline-secondary me-3"
          onClick={[Function]}
        >
          Logout
        </button>
      </ul>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div>
      <h1
        className="animate__animated animate__hinge"
      >
        DcScreen
      </h1>
      <hr />
      <div
        className="row rows-cols-1 row-cols-md-3 g-3 text-dark"
      >
        <div
          className="col"
        >
          <a
            href="/Hero/dc-batman"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Batman"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-batman.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Batman
                    </h5>
                    <p
                      className="card-text"
                    >
                      Bruce Wayne
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Detective Comics #27
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-superman"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Superman"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-superman.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Superman
                    </h5>
                    <p
                      className="card-text"
                    >
                      Kal-El
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Action Comics #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-flash"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Flash"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-flash.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Flash
                    </h5>
                    <p
                      className="card-text"
                    >
                      Jay Garrick
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Flash Comics #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-green"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Green Lantern"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-green.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Green Lantern
                    </h5>
                    <p
                      className="card-text"
                    >
                      Alan Scott
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        All-American Comics #16
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-arrow"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Green Arrow"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-arrow.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Green Arrow
                    </h5>
                    <p
                      className="card-text"
                    >
                      Oliver Queen
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        More Fun Comics #73
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-wonder"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Wonder Woman"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-wonder.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Wonder Woman
                    </h5>
                    <p
                      className="card-text"
                    >
                      Princess Diana
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        All Star Comics #8
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-martian"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Martian Manhunter"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-martian.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Martian Manhunter
                    </h5>
                    <p
                      className="card-text"
                    >
                      J'onn J'onzz
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Detective Comics #225
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-robin"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Robin/Nightwing"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-robin.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Robin/Nightwing
                    </h5>
                    <p
                      className="card-text"
                    >
                      Dick Grayson
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Detective Comics #38
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-blue"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Blue Beetle"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-blue.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Blue Beetle
                    </h5>
                    <p
                      className="card-text"
                    >
                      Dan Garret
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Mystery Men Comics #1
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
        <div
          className="col"
        >
          <a
            href="/Hero/dc-black"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "none",
              }
            }
          >
            <div
              className="card bg-secondary text-white h-100"
            >
              <div
                className="row no-gutters"
              >
                <div
                  className="container-img"
                >
                  <img
                    alt="Black Canary"
                    className="h-100 w-100 animate__animated animate__zoomIn"
                    src="/assets/dc-black.jpg"
                  />
                </div>
                <div
                  className="col-8 w-100"
                >
                  <div
                    className="card-body"
                  >
                    <h5
                      className="card-title"
                    >
                      Black Canary
                    </h5>
                    <p
                      className="card-text"
                    >
                      Dinah Drake
                    </p>
                    <p
                      className="card-text"
                    >
                      <small
                        className="text-white-50"
                      >
                        Flash Comics #86
                      </small>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>,
]
`;
